src/Microsoft.Data.Sqlite/Interop/MarshalEx.cs

```diff
@@ -64,7 +64,7 @@ public static void ThrowExceptionForRC(int rc, Sqlite3Handle db)
                 ? NativeMethods.sqlite3_errstr(rc)
                 : NativeMethods.sqlite3_errmsg(db);
 
-            throw new SqliteException("Error: " + message, rc);
+            throw new SqliteException(Strings.FormatSqliteNativeError(rc, message), rc);
```

Comment:
My reasoning for this was that it's the format returned by sqlite3.exe. I didn't bother with a resource since SQLite itself isn't localized.